models/Recipe.model.js

```diff
@@ -2,7 +2,38 @@ const mongoose = require('mongoose');
 const Schema = mongoose.Schema;
 
 const recipeSchema = new Schema({
-  // TODO: write the schema
+  title : {type:String,required:true},
+  level :{
+          type:String,
```

Comment:
check indentation here, you can also use the auto format feature of VScode with prettier to help with this